src/client/lazy-app/icons/index.tsx

```diff
@@ -11,6 +11,36 @@ const Icon = (props: preact.JSX.HTMLAttributes) => (
   />
 );
 
+export const InfoIcon = (props: preact.JSX.HTMLAttributes) => (
+  <svg viewBox="0 0 512 512" fill="currentColor">
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M64 0C28.6538 0 0 28.6538 0 64V448C0 483.346 28.6538 512 64 512H448C483.346 512 512 483.346 512 448V64C512 28.6538 483.346 0 448 0H64ZM297.543 377.307C298.628 376.1 299.756 374.845 300.942 373.528C305.12 368.875 306.716 364.537 304.731 357.714C302.602 350.359 293.989 346.698 286.361 349.472C276.039 353.254 258.652 363.899 258.652 363.899C258.652 363.899 271.846 302.925 272.394 300.272C273.405 295.411 274.754 290.186 276.158 284.752C282.481 260.273 289.898 231.564 272.33 212.775C257.395 196.816 238.201 201.114 221.267 217.299C211.391 226.739 205.608 233.171 197.254 242.461C196.152 243.686 195.006 244.961 193.8 246.298C189.622 250.959 188.042 255.298 190.026 262.12C192.155 269.483 200.767 273.136 208.38 270.354C218.702 266.579 236.089 255.935 236.089 255.935C236.089 255.935 228.041 293.038 224.009 311.602C223.965 311.814 223.781 312.453 223.491 313.463C220.145 325.103 202.659 385.926 222.412 407.051C237.347 423.019 256.557 418.712 273.475 402.535C283.372 393.074 289.159 386.636 297.543 377.307ZM226.138 136.656C228.235 160.559 249.283 178.219 273.169 176.123C297.039 174.043 314.716 152.979 312.62 129.1C310.539 105.222 289.475 87.5449 265.605 89.6419C241.718 91.7219 224.058 112.778 226.138 136.656Z"
+    />
+  </svg>
```

Comment:
Passed it through SVGOMG.

```suggestion
  <svg viewBox="0 0 512 512" fill="currentColor">
    <path fill-rule="evenodd" d="M64 0C29 0 0 29 0 64v384c0 35 29 64 64 64h384c35 0 64-29 64-64V64c0-35-29-64-64-64H64Zm234 377 3-3c4-5 6-9 4-16-2-8-11-11-19-9-10 4-27 15-27 15l13-64 4-15c6-25 14-53-4-72-15-16-34-12-51 4l-24 25-3 4c-4 5-6 9-4 16s11 11 18 8c11-3 28-14 28-14l-12 56-1 1c-3 12-20 73-1 94 15 16 35 12 51-4l25-26Zm-72-240a43 43 0 1 0 87-8 43 43 0 0 0-87 8Z"/>
  </svg>
```